index.js

```diff
@@ -58,23 +58,25 @@ function processArgv(argv) {
 
   // Look globally and locally for matching glob pattern
   const dirs = [
-    `${process.cwd()}`, // local (where they ran the command from)
-    `${__dirname}/..`   // global (where this code exists)
+    `${process.cwd()}/node_modules`, // local (where they ran the command from)
+    `${__dirname}/..`,  // global, if scoped package (where this code exists)
+    `${__dirname}/../..`, // global, if not scoped package
   ];
 
   getGlobs(dirs).forEach(pkg => {
-    let module;
+    const dirName = path.dirname(pkg);
     let pkgJson = {};
+    let module;
 
     try {
-      module = require(path.dirname(pkg));
+      module = require(dirName);
       pkgJson = require(pkg);
     } catch (err) {
-      logger.error(`Error loading module: ${pkg}`);
+      logger.info(`Error loading module: ${pkg}`);
```

Comment:
Why info vs error?

Reply:
Good question.  After seeing it in practice, using an `error` for this scenario seemed heavy-handed.  While it is possible an actual error has occurred, the most common use-case for this scenario is simply having a matching project name `skyux-builder-*`, which is already somewhat common, and not having a `main` property / `index.js` file.

Reply:
Ah, that makes sense to me. The plugins would be a good example.